### DISCUSSION TITLE
Issue using Highstock with Nuxt3

### DISCUSSION THREAD

**Discussion by rhoudroge** [richardeschloss/nuxt-highcharts#67] [Show and tell]
Hi!

Thanks for the efforts in making and maintaining this library.

I am running into two weird issues, interestingly enough this is not working only in production (it is working in my dev environment): 
- the highstock component not working
- the exporting modules is not working

Both raise the same type of message:

```
TypeError: x.pushUnique is not a function
    at w.compose (exporting.31edbdb5.js:1:3704)
    at K.compose (exporting.31edbdb5.js:1:18605)
    at exporting.31edbdb5.js:1:19408
    at N (exporting.31edbdb5.js:1:572)
    at exporting.31edbdb5.js:1:19196
    at entry.bd4fb60f.js:12:311422
    at async Promise.all (/index 0)
    at async Proxy.mounted (entry.bd4fb60f.js:12:311506)
```

```
entry.fa511ed8.js:1 TypeError: G.pushUnique is not a function
    at k.compose (stock.5e135750.js:1:2854)
    at stock.5e135750.js:1:5793
    at _ (stock.5e135750.js:1:572)
    at stock.5e135750.js:1:5684
    at entry.fa511ed8.js:12:311422
    at async Promise.all (/index 0)
    at async Proxy.mounted (entry.fa511ed8.js:12:311506)

```

The only related issue I found is on Highcharts: https://github.com/highcharts/highcharts/issues/18893.

In both cases, this is my nuxt.config.ts file:

```
  highcharts: {
    exporting: true
  },
  modules: [
    '@pinia/nuxt',
    'nuxt-highcharts'
  ],
```

Would you happen to know how I can fix this ? 

Thanks!
Rami

**Comment by richardeschloss** [comment]
Can you please provide more information? Are you able to provide component usage or a minimal "production-like" repro in a stackblitz?

**Reply by rhoudroge** [reply]
Hi Richard, thanks for answering. I found another issue in the highcharts github repo:  (https://github.com/highcharts/highcharts/issues/18902). I think it might be the source of the problem I am encountering. I will open the issue there and let you know.

**Reply by rhoudroge** [reply]
FYI: https://github.com/highcharts/highcharts/issues/19199

**Reply by richardeschloss** [reply]
I just pushed nuxt-highcharts tested against v11:

> npm i nuxt-highcharts@next

**Reply by rhoudroge** [reply]
Thanks! I will give it a try ASAP and let you know!

**Reply by rhoudroge** [reply]
Awesome, works like a charm, thanks!
